### ISSUE TITLE
Updating migrations table using Oracle fails

### ISSUE THREAD

**Issue by bobvandevijver** [doctrine/migrations#788]
### Bug Report

|    Q        |   A
|------------ | ------
| BC Break    | no
| Version     | 2.0

#### Summary

The new `executed_at` column as introduced in !675 (awesome feature btw!) unfortunately breaks migrations on Oracle databases as the SQL statement generated to update the migrations table is incorrect. 
```
ALTER TABLE MIGRATION_VERSIONS ADD (version VARCHAR2(14) NOT NULL, executed_at TIMESTAMP(0) DEFAULT NULL NULL)
```

As you can see, it tries to re-add the already existing table (which is missing the `executed_at` column), which obviously then fails with:

```
ORA-01430: column being added already exists in table
```

Note we've also just updated several MariaDB and Postgres servers without any issue, so it seems that only Oracle is affected.

#### Current behavior

Migrations are broken when using an Oracle database (until manual intervention)

#### How to reproduce

Remove the `executed_at` table if you have one, and run (using the Symfony migrations bundle for integration):

```
php bin/console doctrine:migrations:status
```

#### Expected behavior

No errors 😄

**Comment by jwage** [comment]
Why does that column already exist? Did you add it manually. I don't have a way to test against Oracle easily. Can you help triage the problem?

**Comment by bobvandevijver** [comment]
I'm afraid you read the message wrong: Oracle warns about the `version` column already being there (which is and should be, as we upgraded from 1.8). The generated SQL to add the `executed_at` column also tries to add the `version` column, which is indeed invalid.

I can test, but I unfortunately do not have much time to spare to find the actual cause.

**Comment by SenseException** [comment]
@bobvandevijver Which version of Oracle DB is in use?

**Comment by bobvandevijver** [comment]
I've now seen on these two versions:
* Oracle Database 12c Enterprise Edition Release 12.1.0.2.0 - 64bit Production
* Oracle Database 11g Express Edition Release 11.2.0.2.0 - 64bit Production

**Comment by jwage** [comment]
If you manually add the new columns to the table, does Doctrine stop trying to update the table?

**Comment by bobvandevijver** [comment]
Yes, if I manually add the `executed_at` column it stops complaining and it then works as I would expect. Tried mentioning that in the original post, but that might not have been as clear as I thought :)

**Comment by hmonglee** [comment]
Hello I have the same issue with Oracle. I made a PR.
